### ISSUE TITLE
Some troubles with codes

### ISSUE THREAD

**Issue by M-SunRise** [Leminhbinh0209/ADD#5]
I met some troubles when I try to run it.

in the train.py line54:
`from distiller_zoo import Attention, HintLoss, Correlation, VIDLoss, RKDLoss,  PKT, DistillKL, Frequency, AttentionPx, Multiview, NonLocal`
I couldn't find the "DistillKL"、“AttentionPx”,"VIDLoss"。

in the train.py line214:   `criterion_div = DistillKL(config.kd_T)`     the same problem I met.

config.is_proj and config.is_freq mean?

**Comment by Leminhbinh0209** [comment]
They are just distiller modules. since I don't use them in my paper, you can remove them in code.

**Comment by M-SunRise** [comment]
`Train Epoch: 03 Batch: 2873/2873 | L_F: 0.0000 | Loss_KD: 9.2190 |  Loss_CE: 0.3983 | Loss:147.9029 | Acc: 0.8182 | AUC: 0.8997`
`Train Epoch: 04 Batch: 0001/2873 | L_F: 14491869.0000 | Loss_KD: 9.4722 |  Loss_CE: 0.4071 | Loss:14492021.0000 | Acc: 0.8056 |`
`Train Epoch: 04 Batch: 0038/2873 | L_F: 1293452.3257 | Loss_KD: 9.1981 |  Loss_CE: 0.5173 | Loss:1293599.9910 | Acc: 0.7403 |`
the L_F in the 1,2,3 epoch is 0, in the epoch 4, it increases to 14491869.0000, and the acc decent quickly. It maybe have some problems. Do you have met?

**Comment by Leminhbinh0209** [comment]
Hi, I have not encountered it before, but you can fine-tune the hyper-parameter. And make sure you have torch.rfft produce shape of B x C x W x H x 2,  I see some later updates produce a different size.

**Comment by M-SunRise** [comment]
I read the code, and found that the teacher model's inputs are raw images, and the student model's inputs are compressed images. In paper, Table 1, how could you deal with the raw and compressed images？ In the code, there is JPEGcompression function. But I don't know which values of compression_qf could represent c23, c40.
I tried to solve it in the another way, to crop the raw and compressed images form FF++ videos, respectively. However it maybe more time consuming. Do you have any methods to solve this problem. 
Thank you for your positive response and good luck!

**Comment by Leminhbinh0209** [comment]
Hi, 
I planned to apply compression_qf for those datasets that don't have a compression version (e.g., DFDC, ...), but I hadn't experimented yet. So you can ignore this param. And for the FF++, you have to use ```img_com = Image.open(img_path.replace('raw', self.quality)) ```. where  self.quality = c23 or c40.
I did the face detection on raw dataset, and use the coordinate of bounding box for both raw, and c23, c40. This can save you a lot of time and the model can only run correctly in this manner.

**Comment by M-SunRise** [comment]
Thanks for your reply！

**Comment by Leminhbinh0209** [comment]
Hi,
Yes, the program can be unstable. Therefore, you can set the random seed in the ```main.py```:
```    
torch.manual_seed(random_seed)
torch.cuda.manual_seed(random_seed)
torch.cuda.manual_seed_all(random_seed)
torch.backends.cudnn.deterministic = True
torch.backends.cudnn.benchmark = False
np.random.seed(random_seed)
```
and get the average of multi running.
